elasticsearch/azuredeploy.parameters.json

```diff
@@ -3,13 +3,13 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "dnsNameforLBIP": {
-      "value": "changeme"
+      "value": "elasticlbip"
     },
     "storageAccountPrefix": {
-      "value": "changeme"
+      "value": "elasticstore"
     },
     "adminUsername": {
-      "value": "trent"
+      "value": "admin"
```

Comment:
This username might be blocked on the platform, will cause the automated bot deployment to fail